### ISSUE TITLE
service module does not enable correctly via update-rc.d 

### ISSUE THREAD

**Issue by msolo** [ansible/ansible-modules-core#90]
##### Issue Type:

“Bug Report”
##### Ansible Version:

ansible 1.8
##### Environment:

Ubuntu 13.10/14.04
##### Summary:

Fix or remove the faulty change detection for update-rc.d based service enabling.
##### Steps To Reproduce:

```
sudo apt-get install rrdcached
```

Correctly enabled after installation:

```
find /etc -iname '*rrdcached*' | sort
/etc/default/rrdcached
/etc/init.d/rrdcached
/etc/rc0.d/K20rrdcached
/etc/rc1.d/K20rrdcached
/etc/rc2.d/S20rrdcached
/etc/rc3.d/S20rrdcached
/etc/rc4.d/S20rrdcached
/etc/rc5.d/S20rrdcached
/etc/rc6.d/K20rrdcached
```

Disable:

```
ansible localhost -s -m service -a "name=rrdcached enabled=no"
localhost | success >> {
    "changed": true, 
    "enabled": false, 
    "name": "rrdcached"
}
```

Correctly disabled:

```
find /etc -iname '*rrdcached*' | sort
/etc/default/rrdcached
/etc/init.d/rrdcached
/etc/rc0.d/K20rrdcached
/etc/rc1.d/K20rrdcached
/etc/rc2.d/K80rrdcached
/etc/rc3.d/K80rrdcached
/etc/rc4.d/K80rrdcached
/etc/rc5.d/K80rrdcached
/etc/rc6.d/K20rrdcached
```

Re-enable:

```
ansible localhost -s -m service -a "name=rrdcached enabled=yes"
localhost | success >> {
    "changed": false, 
    "enabled": true, 
    "name": "rrdcached"
}
```

Note: Changed is false, which is incorrect.

Incorrect results on disk too:

```
find /etc -iname '*rrdcached*' | sort
/etc/default/rrdcached
/etc/init.d/rrdcached
/etc/rc0.d/K20rrdcached
/etc/rc1.d/K20rrdcached
/etc/rc2.d/K80rrdcached
/etc/rc3.d/K80rrdcached
/etc/rc4.d/K80rrdcached
/etc/rc5.d/K80rrdcached
/etc/rc6.d/K20rrdcached
```
##### Expected Results:

```
find /etc -iname '*rrdcached*' | sort
/etc/default/rrdcached
/etc/init.d/rrdcached
/etc/rc0.d/K20rrdcached
/etc/rc1.d/K20rrdcached
/etc/rc2.d/S20rrdcached
/etc/rc3.d/S20rrdcached
/etc/rc4.d/S20rrdcached
/etc/rc5.d/S20rrdcached
/etc/rc6.d/K20rrdcached
```
##### Actual Results:

```
find /etc -iname '*rrdcached*' | sort
/etc/default/rrdcached
/etc/init.d/rrdcached
/etc/rc0.d/K20rrdcached
/etc/rc1.d/K20rrdcached
/etc/rc2.d/K80rrdcached
/etc/rc3.d/K80rrdcached
/etc/rc4.d/K80rrdcached
/etc/rc5.d/K80rrdcached
/etc/rc6.d/K20rrdcached
```

**Comment by jonhadfield** [comment]
I can reproduce this specific issue up to v1.8.4 but not after service.py updates in f4a709a (checked out as submodule of current devel branch).

**Comment by bcoca** [comment]
I'm going to close this ticket then as I expect f4a709a to have fixed it and it will be part of the 1.9 release

**Comment by soltiz** [comment]
Not fixed in 1.9
